css/src/tailwind.css

```diff
@@ -1,4 +1,5 @@
 @import "@yoast/ui-library";
+@import "@yoast/related-keyphrase-suggestions";
```

Comment:
This should not be here. We load in this stylesheet if we want to use the UI library or Tailwind.
While the related keyphrase suggestions is only interesting in the context of the editors.